src/EditorFeatures/Core/IInlineRenameSession.cs

```diff
@@ -43,8 +43,12 @@ internal interface IInlineRenameSession
         void Cancel();
 
         /// <summary>
-        /// Dismisses the rename session, completing the rename operation across all files.
+        /// Attempts to commit the rename session. If <paramref name="previewChanges"/> is true or
```

Comment:
The "or" doesn't seem right here -- I'd expect the user setting to always take priority. If I don't want previews, I don't want previews. Ever.